ajax/libs/jquery.i18n/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jquery.i18n",
+  "description": "jQuery.i18n is a jQuery based Javascript internationalization library. It helps you to internationalize your web applications easily.",
+  "filename": "jquery.i18n.js",
+  "keywords": [
+    "jquery",
+    "i18n",
+    "internationalization"
```

Comment:
Add `localization` and `l10n` to correspond https://github.com/wikimedia/jquery.i18n/blob/master/package.json#L6-L11